src/Cake.Gradle.Tests/GradleRunnerFixture.cs

```diff
@@ -1,40 +1,44 @@
-﻿using Cake.Testing.Fixtures;
+﻿using Cake.Core.IO;
+using Cake.Testing.Fixtures;
 
 namespace Cake.Gradle.Tests
 {
     public class GradleRunnerFixture : ToolFixture<GradleRunnerSettings>
     {
-        private GradleLogLevel? _gradleLogLevel;
-        private string _task;
-        private string _arguments;
+        private GradleRunner tool;
```

Comment:
Did we want to follow the `_` convention that is already in place here?

Reply:
If not, we should probably change the others to not have the `_`.

Reply:
@gep13 good point. I planned on enabling that, but somehow it failed and I did not even notice.

Reply:
@gep13 I modified `.editconfig` accordingly and fixed this.